### DISCUSSION TITLE
Online course for learning the use of MIMIC IV from the basis

### DISCUSSION THREAD

**Discussion by elad-chen** [MIT-LCP/mimic-code#1654] [MIMIC-IV]
hey, would somebody knows if there is an online course to learn how to usr the MIMIC IV from the basis?
I need to use this dataset at mu research and don't know how to use.

Thanks for your help!

Elad

**Comment by marymlucas** [comment]
Hi @elad-chen 
I'm not sure about an online course, but @alistairewj has a really helpful tutorial that may provide a good starting point. I believe it's based on a previous version of MIMIC-IV but still very helpful -[MIMIC-IV: Analysis Tutorial](https://alistairewj.github.io/talk/2020-mimic-iv-analysis-tutorial/)


Mary
